ddtrace/internal/writer.py

```diff
@@ -124,6 +125,10 @@ def flush_queue(self):
                                            len(list(grouped_responses)),
                                            tags=['status:%d' % status])
 
+            # Send global stats
+            for stat, val, tags in stats.reset_values():
+                self.dogstatsd.gauge(stat, val, tags=tags)
```

Comment:
I'm pretty sure none of those stats are gauges, right?
They're not absolute, they're increments of something that is measured.

Reply:
yup... you are right.